README.md

```diff
@@ -112,11 +112,16 @@ type NFTPlatform = {
   platformName: string;
   platformLogoUrl: string;
   urlPattern: RegExp;
-  urlExtractor: (url: string) => NFTExtraction | undefined;
+  urlExtractor?: (url: string) => NFTExtraction | undefined;
```

Comment:
Do we need to separate the async and sync logic? Could we have just the async logic and make sure we returned a resolved promise for the sync extractor?

Reply:
Yeah you're right there isn't a benefit to separating so I removed the async function and updated urlExtractor to be compatible